ansible_module/conns/qubes.py

```diff
@@ -85,6 +93,9 @@ def _qubes(self, cmd=None, in_data=None):
         else:
             # For dom0
             local_cmd.extend(["qvm-run", "--pass-io", "--service"])
+            if self.user != "user":
+                # Means we have a remote_user value
+                local_cmd.extend(["-u", self.user])
```

Comment:
I think your changes from #3 got dragged in here :)

Reply:
Yup, this was intentional :)